src/utils/dato.js

```diff
@@ -42,7 +47,21 @@ const vaskInputDato = dato => {
   return returnDate;
 };
 
+/** Normalisering gjennom Redux prop (normaize) sender 2 argumenter. Dersom disse er forskjellige,
+ * indikerer det at brukeren ikke har forlatt skjemafeltet. Normalize kalles altså en ekstra gang onBlur.
+ * Først dersom begge verdiene er like skal normalisering skje.
+ *
+ * @param verdi Totalverdien av feltet ETTER siste tastetrykk
+ * @param forrigeVerdi Totalverdien av feltet FØR siste tastetrykk
+ * @returns {String}
+ */
+const normaliserInputDato = (verdi, forrigeVerdi) => {
```

Comment:
Vi bør normalisere dato så snart den er tastet inn for å gi brukeren en umiddelbar korrekt dato eller en feilmelding.